scripts/release/generate-changelog.js

```diff
@@ -10,11 +10,23 @@ const { spawnCommand, printError, runMain } = require('../lib/execution-utils')
 const { command } = require('../lib/command')
 const { modifyFile } = require('../lib/files-utils')
 
-const CHANGELOG_FILE = 'CHANGELOG.md'
-const CONTRIBUTING_FILE = 'CONTRIBUTING.md'
-const EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚗️', '♻️']
-const START_WITH_EMOJI = /^\p{Emoji_Presentation}/u
-
+const CHANGELOG_FILE = '../../CHANGELOG.md'
+const CONTRIBUTING_FILE = '../../CONTRIBUTING.md'
+const PUBLIC_EMOJI_PRIORITY = ['💥', '✨', '🐛', '⚡️', '📝', '⚗️']
```

Comment:
⚠️  The ⚡️ emoji in that list is incorrect. It's encoded as `'\u26a1\ufe0f'` but should be only `'\u26a1'`

Reply:
good catch 👍  I will update this in the next Pr